index.html

```diff
@@ -7,123 +7,131 @@
     <title>Sunga's Portfolio</title>
     <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-<section class="Home"></section>    
-<header class="mob-home-header">
-    <div class="name logo">
-        <div class="name-text" ><a href="" id="logo">Sungabanja Thawethe</a></div>
-    </div>
-    <nav>
+
+  <section class="Home">   
+    <header class="mob-home-header">
+      <div class="name logo">
+        <div class="name-text" ><a href="" id="logo">Sunga</a></div>
+      </div>
+      <nav>
+        
+        <div class="mob-nav-list">
+          <div class="close-icon"><img src="images/close.png"></div>
+          <ul>
+          <li><a href="#Portfolio">Portfolio</a></li>
+          <li><a href="#About">About</a></li>
+          <li><a href="#Contact">Contact</a></li>
+          </ul>
+          </div>
         <div class="name menuicon">
-        <img class="hamburger" src="./images/Hamburger.png" alt="menu icon">
+          <img class="hamburger" src="./images/Hamburger.png" alt="menu icon">
         </div>
-    </nav>
-</header>
```

Comment:
- [ ] On the mobile view I can see a horizontal scroll bar at the bottom of the page meaning that your page is overflowing. Please fix that.
- [ ] When I click on the hamburger button, noting pops up. Please fix that.

Reply:
Kindly elaborate the points brought up. Thanks in advance.